pytorch_lightning/core/lightning.py

```diff
@@ -1326,6 +1326,23 @@ def training_step(...):
         self.trainer.train_loop.backward(loss, optimizer=None, opt_idx=None, *args, **kwargs)
         self._running_manual_backward = False
 
+    def on_before_backward(self, loss: torch.Tensor) -> None:
+        """
+        Override on_before_backward with your own implementation if you need to.
+
+        Args:
+            loss: Loss is already scaled by accumulated grads and possibly scaled by Mixed Precision.
```

Comment:
is it possible someone wants the unscaled loss?

Reply:
I would assume the user to expect the final loss to be used for `backward` or open an issue if they want more.